common/src/main/java/io/pravega/common/SecurityConfigDefaults.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.common;
+
+/**
+ * Holds default security configuration values.
+ */
+public class SecurityConfigDefaults {
```

Comment:
I see these definitions being used in tests and standalone. are those defaults for distributed Pravega too?

Reply:
They are set as the defaults in standalone mode config file. They can also be used in distributed mode, but not configured as defaults in the configuration files used in distributed mode.